java/client/src/main/java/glide/ffi/resolvers/RedisValueResolver.java

```diff
@@ -17,4 +17,13 @@ public class RedisValueResolver {
      * @return A RESP3 value
      */
     public static native Object valueFromPointer(long pointer);
+
+    /**
+     * Resolve a value received from Redis using given C-style pointer. This method does not assume
+     * that strings are valid UTF-8 encoded strings
+     *
+     * @param pointer A memory pointer from {@link Response}
+     * @return A RESP3 value
+     */
+    public static native Object valueFromPointerBinary(long pointer);
```

Comment:
Remove

Reply:
As discussed: once all commands are converted - we will do code cleanup